pytest_plugins/fixture_markers.py

```diff
@@ -6,7 +6,10 @@
 
 
 def pytest_generate_tests(metafunc):
-    if 'rhel_contenthost' in metafunc.fixturenames:
+    content_host_fixture = ''.join(
```

Comment:
This pattern is a bit odd, if both fixtures are used this will end up concatenating their names and failing the check.

I think what you really want to do is this, which will be true if either string is in the fixturenames list.
```
if any([True for f in ['rhel_contenthost', 'content_hosts'] if f in metafunc.fixturenames])
```

Reply:
Hmm...I took it under the impression that only 1 of the fixture will be called at any given point. 
I forgot it also could be `def test(rhel_contenthost, content_hosts)`.

Reply:
Spoke a bit too soon - you want to pick the function name to use in the parametrization below as well.  I'd still avoid joining the function names. You can use comprehension and then just check that the list is non-empty.